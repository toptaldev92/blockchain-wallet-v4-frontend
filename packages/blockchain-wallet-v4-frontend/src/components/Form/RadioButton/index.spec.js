import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import RadioButton from './index.js'

jest.mock('blockchain-info-components', () => {
  return {
    Text: jest.fn(),
    RadioButtonInput: jest.fn()
  }
})

describe('RadioButton', () => {
  it('renders correctly', () => {
    const props = { meta: {} }
    const component = shallow(<RadioButton {...props} />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
