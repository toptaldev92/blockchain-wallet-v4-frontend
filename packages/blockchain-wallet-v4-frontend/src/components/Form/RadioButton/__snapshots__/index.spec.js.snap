// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton renders correctly 1`] = `
<styled.div>
  <styled.div>
    <mockConstructor
      errorState="initial"
      props={
        Object {
          "id": undefined,
        }
      }
    />
  </styled.div>
</styled.div>
`;
